plato/views/views.py

```diff
@@ -33,6 +33,7 @@ class TemplateDetailView(NamedTuple):
     type: str
     metadata: dict
     tags: Sequence[str]
+    example_composition: dict
```

Comment:
small thing, but example composition is missing in the docstring

Reply:
You are completely right! Also noticed it missing on some other docstrings, so took the chance to update them as well.